server.js

```diff
@@ -4,17 +4,17 @@ const config = require("./config/config");
 const Logger = require("./config/logger");
 
 const server = app.listen(config.port, () => {
-    const serverStartTime = new Date().toLocaleString();
-    Logger.log("info", {
-        message: `Backend server started at port ${config.port} on ${serverStartTime}`,
-    });
+  const serverStartTime = new Date().toLocaleString();
```

Comment:
use moment library instead o native Date. change this wherever you have used date